templates/install-wp-tests.sh

```diff
@@ -116,6 +116,7 @@ install_test_suite() {
 		# remove all forward slashes in the end
 		WP_CORE_DIR=$(echo $WP_CORE_DIR | sed "s:/\+$::")
 		sed $ioption "s:dirname( __FILE__ ) . '/src/':'$WP_CORE_DIR/':" "$WP_TESTS_DIR"/wp-tests-config.php
+		sed $ioption "s:__DIR__ . '/src/':'${WP_CORE_DIR}/':" "$WP_TESTS_DIR"/wp-tests-config.php
```

Comment:
Although I prefer the curly brace notation (with the dollar sign within the curly braces), we should opt for consistency for now and change all of the substitutions in one go for style in a separate PR.
```suggestion
		sed $ioption "s:__DIR__ . '/src/':'$WP_CORE_DIR/':" "$WP_TESTS_DIR"/wp-tests-config.php
```

Reply:
Thank you for explanation, Later I will prepare separate PR. Fixed here.